pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1235,8 +1241,11 @@ func (qjm *XController) ScheduleNext() {
 		// HeadOfLine logic
 		HOLStartTime := time.Now()
 		forwarded := false
+		fowardingLoopCount := 1
```

Comment:
@asm582, @dmatch01, @KPostOffice and @metalcycling  please take a very close look at the changes starting from this line.